tools/rdm/TestDefinitions.py

```diff
@@ -6473,14 +6433,177 @@ class GetListInterfacesWithData(TestMixins.GetWithDataMixin,
   CATEGORY = TestCategory.ERROR_CONDITIONS
   PID = 'LIST_INTERFACES'
 
-class SetListInterfaces(ResponderTestFixture):
+class SetListInterfaces(TestMixins.UnsupportedSetMixin,
+                        OptionalParameterTestFixture):
   """Attempt to SET list interfaces."""
   CATEGORY = TestCategory.ERROR_CONDITIONS
   PID = 'LIST_INTERFACES'
 
+# DNS_HOSTNAME
+#------------------------------------------------------------------------------
+class GetDNSHostname(TestMixins.GetStringMixin,
+                     OptionalParameterTestFixture):
+  """GET the DNS hostname."""
+  CATEGORY = TestCategory.IP_DNS_CONFIGURATION
+  PID = 'DNS_HOSTNAME'
+  EXPECTED_FIELD = 'dns_hostname'
+  ALLOWED_NACK = RDMNack.NR_HARDWARE_FAULT
+  MIN_LENGTH = RDM_MIN_HOSTNAME_LENGTH
+  MAX_LENGTH = RDM_MAX_HOSTNAME_LENGTH
+
+class GetDNSHostnameWithData(TestMixins.GetWithDataMixin,
+                             OptionalParameterTestFixture):
+  """Get DNS hostname with param data."""
+  CATEGORY = TestCategory.ERROR_CONDITIONS
+  PID = 'DNS_HOSTNAME'
+  # Allow NR_HARDWARE_FAULT in case they're checking length then PDL
+  ALLOWED_NACK = RDMNack.NR_HARDWARE_FAULT
```

Comment:
Looks like you already fixed it - that's what I would have suggested . I think we can remove it now, thanks.